utility/unit_of_work/gov.go

```diff
@@ -3,12 +3,108 @@ package unit_of_work
 import (
 	"math/big"
 
+	"github.com/pokt-network/pocket/persistence"
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
 	"github.com/pokt-network/pocket/shared/utils"
 	typesUtil "github.com/pokt-network/pocket/utility/types"
 	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
+func init() {
+	govParamTypes = prepareGovParamParamTypesMap()
+}
+
+var (
+	govParamTypes map[string]int
+)
+
+const (
+	BIGINT int = iota
+	INT
+	INT64
+	BYTES
+	STRING
+)
+
+func prepareGovParamParamTypesMap() map[string]int {
```

Comment:
Regarding this comment: https://github.com/pokt-network/pocket/pull/622/files/010c045d060cf8114d09066846c1f7d1fe5f7318..2158a38a6ff9529a405a23c55366e37d403afa5b#r1160332826

If we update `genesis.proto`, there's nothing currently that forces us to update `prepareGovParamParamTypesMap` as well.

Source of truth: `genesis.proto`
Derived source of truth based on above: `GovParamMetadataMap`
Alternate source of truth: `prepareGovParamParamTypesMap`

In the forloop, we currently ignore missing ids: `switch id := govParamTypes[paramName]; id {`

tl;dr I basically want a fatal error if the ID doesn't exist.

Reply:
Well this will always result in an error as the main map has the owners in it too, but this type map doesn't. I will add some logic to the init where it loops over the non-owner keys and checks they have a type in the map

Reply:
Good think we did this I missed 2 😅

Reply:
Paying dividends on day 0.